test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -43,12 +53,33 @@ describe("BENEFICIAL OWNER GOV controller", () => {
   describe("POST tests", () => {
 
     test("redirects to the managing-officer page", async () => {
+      mockPrepareData.mockReturnValueOnce(BENEFICIAL_OWNER_GOV_OBJECT_MOCK);
       const resp = await request(app).post(config.BENEFICIAL_OWNER_GOV_URL);
 
       expect(resp.status).toEqual(302);
       expect(resp.header.location).toEqual(config.MANAGING_OFFICER_URL);
     });
 
+    test("adds data to the session", async () => {
+      mockPrepareData.mockReturnValueOnce({ name: "test" });
```

Comment:
In other tests, rather than mock all the return values we've done:

`mockPrepareData.mockImplementationOnce( () => BENEFICIAL_OWNER_GOV_OBJECT_MOCK );
`
Less lines of code. Any reason it's being done differently here?

Reply:
it wasn't really an accurate reflection of how the controller built the data (see previous comment for more detailed explanation)